android-gif-drawable/src/test/java/pl/droidsonroids/gif/GifViewUtilsDensityTest.kt

```diff
@@ -0,0 +1,241 @@
+package pl.droidsonroids.gif
+
+import android.content.res.Resources
+import android.graphics.Bitmap
+import android.os.Build
+import android.util.DisplayMetrics
+import android.util.TypedValue
+import androidx.annotation.RequiresApi
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.ArgumentMatchers
+import org.mockito.Mock
+import org.mockito.Mockito
+import org.mockito.junit.MockitoJUnitRunner
+import pl.droidsonroids.gif.GifViewUtils.getDensityScale
+
+@RequiresApi(Build.VERSION_CODES.JELLY_BEAN_MR2)
+@RunWith(
+    MockitoJUnitRunner::class
+)
```

Comment:
```suggestion
@RunWith(MockitoJUnitRunner::class)
```
Change also similar places, if any.